Update items.csv

clou44 opened the pull request:
Added shorter aliases for salmon, pufferfish, and clownfish after input from players.

clou44 committed:
Update items.csv

Added shorter aliases for salmon, pufferfish, and clownfish after input from players.

necrodoom commented on Essentials/src/items.csv:
```diff
@@ -4579,15 +4579,25 @@ salmonfish,349,1
 nemo,349,2
+clfish,349,2
+fishcl,349,2
+nfish,349,2
```
add nemofish and fishnemo for consistency then please

clou44 committed:
Update items.csv

Added nemofish and fishnemo. Didn't notice that they weren't there, to be honest.

khobbits commented:
merged.
